### PR TITLE
Switch to Github Actions

### PR DESCRIPTION
**Pull request by simonschaufi**
Inspired by https://github.com/laravel/socialite/commit/8b1cf10485c64ae3033cf5c2b6b12b0699d899c3 and https://github.com/spatie/laravel-tags/blob/b22dd237ea795e0bc0613643c222e4ce23f52859/.github/workflows/run-tests.yml

Adjusted the php linter. Source from TYPO3 Core: https://github.com/TYPO3/TYPO3.CMS/blob/960090d692ed77080e2d6453e47dd8bdef1498c0/Build/testing-docker/local/docker-compose.yml#L833

### CODE REVIEW

**Comment by simonschaufi** [comment]
I thought this PR would already trigger the tests but unfortunately not. It seams like this needs to get merged in order for following PRs to trigger the tests. I don't know yet how to do database tests.

**Comment by simonschaufi** [comment]
@nicodh will you merge or what's the next step?

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+jobs:
+  tests:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-latest]
+        php: [7.2, 7.3]
```
Comment (liayn) [inline .github/workflows/tests.yml:13]: Why not 7.4? it's officially supported for T3 8-10

Comment (simonschaufi) [reply .github/workflows/tests.yml:13]: I just migrated exactly the settings from travis without any (more or less) adjustments.

**Pull request merged**